tests/test_commands.py

```diff
@@ -626,11 +626,11 @@ def test_client_unpause(self, r):
         assert r.client_unpause() == b"OK"
 
     @pytest.mark.onlynoncluster
-    # @skip_if_server_version_lt("7.0.0") turn on after redis 7 release
-    def test_client_no_evict(self, unstable_r):
-        assert unstable_r.client_no_evict("ON") == "OK"
+    @skip_if_server_version_lt("7.0.0")
+    def test_client_no_evict(self, r):
```

Comment:
Maybe it's time to move our docker to use the unstable one, instead of 6.2.6 Thoughts?

Reply:
Yes, I think we can try it